frontend/src/components/ContextMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import {
+  EuiButton,
+  EuiContextMenuPanel,
+  EuiContextMenuItem,
+  EuiPopover,
+  EuiCopy,
+  useGeneratedHtmlId, EuiButtonEmpty,
+} from '@elastic/eui';
+import {Recording, useDeleteRecordingById} from "../api";
+import {useQueryClient} from "@tanstack/react-query";
+
+export default ({recordingId, deleteCallback}: {recordingId: string, deleteCallback: any}) => {
```

Comment:
Not strictly necessary, but a good practice is to extract the type definition into a separate props type. Then you can write
`export default ({recordingId, deleteCallback}: ContextMenuProps) =>`